Index: mt76-2021-12-02-929a03a8/mac80211.c
===================================================================
--- mt76-2021-12-02-929a03a8.orig/mac80211.c
+++ mt76-2021-12-02-929a03a8/mac80211.c
@@ -425,7 +425,7 @@ mt76_phy_init(struct mt76_phy *phy, stru
 	ieee80211_hw_set(hw, SUPPORTS_RC_TABLE);
 	ieee80211_hw_set(hw, SUPPORT_FAST_XMIT);
 	ieee80211_hw_set(hw, SUPPORTS_CLONED_SKBS);
-	ieee80211_hw_set(hw, SUPPORTS_AMSDU_IN_AMPDU);
+//	ieee80211_hw_set(hw, SUPPORTS_AMSDU_IN_AMPDU);
 	ieee80211_hw_set(hw, SUPPORTS_REORDERING_BUFFER);
 	ieee80211_hw_set(hw, TX_AMSDU);
 	ieee80211_hw_set(hw, TX_FRAG_LIST);
